content/fr/agent/versions/upgrade_to_agent_v6.md

```diff
@@ -324,17 +328,19 @@ Vous trouverez ci-dessous les instructions d'installation manuelle pour :
   type=rpm-md
   gpgcheck=1
   repo_gpgcheck=0
-  gpgkey=https://yum.datadoghq.com/DATADOG_RPM_KEY.public
-         https://yum.datadoghq.com/DATADOG_RPM_KEY_E09422B3.public
-         https://yum.datadoghq.com/DATADOG_RPM_KEY_20200908.public
+  gpgkey=https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public
+         https://keys.datadoghq.com/DATADOG_RPM_KEY_FD4BF915.public
+         https://keys.datadoghq.com/DATADOG_RPM_KEY_E09422B3.public
+         https://keys.datadoghq.com/DATADOG_RPM_KEY.public
   ```
 
 2. Mettez à jour votre référentiel Zypper local et installez l'Agent :
   ```
   sudo zypper refresh
-  sudo rpm --import https://yum.datadoghq.com/DATADOG_RPM_KEY.public
-  sudo rpm --import https://yum.datadoghq.com/DATADOG_RPM_KEY_E09422B3.public
-  sudo rpm --import https://yum.datadoghq.com/DATADOG_RPM_KEY_20200908.public
+  sudo rpm --import https://keys.datadoghq.com/DATADOG_RPM_KEY.public
+  sudo rpm --import https://keys.datadoghq.com/DATADOG_RPM_KEY_E09422B3.public
+  sudo rpm --import https://keys.datadoghq.com/DATADOG_RPM_KEY_FD4BF915.public
+  sudo rpm --import https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public
```

Comment:
Why are these in reverse order? No issue with it (other than maybe harder to search & replace), just curious 😄

Reply:
The order doesn't really matter here, unlike with repofile, where dnf/yum/zypper will go line by line until it finds a key currently used and then stops - so in that case we want to ensure that the future keys get imported on the first run (putting them higher in the list). This place just tells the user to run all of these commands, so we don't care about the order.

But you're right it's slightly confusing to have the order reversed, so I'll change it.